cobalt/site/docs/reference/starboard/modules/13/ui_navigation.md

```diff
@@ -268,6 +268,7 @@ right and top-to-bottom.
 ```
   Top-to-bottom is similar to left-to-right, but for the Y position.
   Bottom-to-top is similar to right-to-left, but for the Y position.
+
```

Comment:
This also looks a little funky to me.

Reply:
can you re-test the devsite generation instructions and see if see if you get the same results?